backend/src/core/models/game.spec.js

```diff
@@ -201,6 +203,46 @@ describe("Game", function () {
     });
   });
 
+  describe(".fromSteamApi", function () {
+    describe("when the method is called", function () {
+      beforeAll(function () {
+        this.result = Game.fromSteamApi(eldenRingSteamApiData);
+      });
+
+      it("is an instance of Game", function () {
+        expect(this.result).toBeInstanceOf(Game);
+      });
+
+      it("the game has the correct values", function () {
+        expect(this.result.id).toBe(1245620);
```

Comment:
create `expectedResult` object in before block with all the expected data and then use here `expect(this.result).toEqual(this.expectedResult)` instead of the many lines